ami/main/models.py

```
@@ -805,11 +806,27 @@ def sync_captures(self, batch_size=1000, regroup_events_per_batch=False, job: "J
             logger.debug(f"Processing file {file_index}: {obj}")
             if not obj:
                 continue
-            source_image = _create_source_image_for_sync(deployment, obj)
+
+            try:
+                source_image = _create_source_image_for_sync(deployment, obj)
+            except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)